mods/ts/rules/civilian-structures.yaml

```diff
@@ -1261,7 +1263,6 @@ GALITE:
 		Range: 2c0
 	Power:
 		Amount: 0
-# TODO: should use terrain lighting instead, depends on https://github.com/OpenRA/OpenRA/issues/3605
```

Comment:
This probably slipped in after running the upgrade rules, please revert this removal.